src/types/view/index.ts

```diff
@@ -7,7 +7,7 @@ import { RespondArguments, AckFn } from '../utilities';
 export interface SlackViewMiddlewareArgs {
```

Comment:
is it worth considering adding a generic parameter for the body type here?

consider the way `app.actions(...)` is used. if you know that the string or constraints you passed in will only ever match a block action, you can use `app.actions<BlockAction>(...)` and in the listener you get a strong type for `action`. 

if we made this a generic type, we might be able to offer the same advantage for when you know you're matching only a submit versus only a closed.

another idea: we could make the default for the generic parameter be the submit type, so that unless you specify it as the union of both, the listener gets the submit as default shape of body.